variables.tf

```diff
@@ -584,30 +584,24 @@ variable "services" {
     family       = optional(string)
     ipc_mode     = optional(string)
     memory       = optional(number, 2048)
-    network_mode = optional(string, "awsvpc")
+    network_mode = optional(string)
```

Comment:
I like that there are fewer default values in "optional()". It is easier to read, and hopefully it also works more predictably. 👍